### PR TITLE
feat(resend-verification): Add Resend Verification endpoint

### PR DESCRIPTION
**Pull request by Daffadon**
This PR contain adding resend new link for verification to user's email address. The endpoint working with some steps:

- Check Email that want to user's reset token
- Checking Email that user's send is already verified or not
- Generating new token and send the link to email address

### CODE REVIEW

File: src/service/token/verification.ts
Comment (Reqyan) [inline src/service/token/verification.ts:1]: Pleas remove the comment that not used by code

**Review by Reqyan — CHANGES_REQUESTED** [review]
Please check the comment in verification.ts

**Comment by Daffadon** [comment]
> Please check the comment in verification.ts

Thank you for your review. I've already delete the useless comment in verification.ts

**Review by Reqyan — APPROVED** [review]
LGTM

**Pull request merged**